packages/react/src/components/Flex/Flex.tsx

```diff
@@ -90,20 +90,27 @@ class Flex extends UIComponent<ReactProps<FlexProps>> {
   renderChildren = (gapClasses: string) => {
     const { column, gap, children } = this.props
 
-    return React.Children.map(children, (child: React.ReactElement<any>, index) => {
-      const childElement =
-        child.type && ((child.type as any) as typeof FlexItem).__isFlexItem
+    let isFirstElement = true
+    return React.Children.map(children, (child: any) => {
+      const maybeChildElement =
+        child && child.type && ((child.type as any) as typeof FlexItem).__isFlexItem
```

Comment:
addressed failures when `child` is `null` - happens when primitive values (other than `string`) are passed

Reply:
```jsx
const isFlexItem = child && child.type && ((child.type as any) as typeof FlexItem).__isFlexItem
// or 
const isFlexItem: boolean = _.get(child, 'type.__isFlexItem')
```